src/components/ui/Box.tsx

```diff
@@ -16,6 +15,32 @@ export interface BoxProps {
   disabled?: boolean;
 }
 
+// scoped to this component?
+Decimal.set({
```

Comment:
Applied.

Also replaced `toInteger()` with `toDecimalPlaces(0, Decimal.ROUND_FLOOR)` since toInteger uses the default `rounding` which is NOT `ROUND_FLOOR`